src/editor/main.cpp

```diff
@@ -303,19 +303,19 @@ LRESULT CALLBACK wnd_proc_static(HWND hWnd, UINT message, WPARAM wParam, LPARAM
     }
     break;
   default:
-    if (message == wm_copydata_webview) {
+    if (message == wm_data_for_webview) {
       PCOPYDATASTRUCT msg = (PCOPYDATASTRUCT)lParam;
       if (msg->dwData == SEND_TO_WEBVIEW_MSG) {
         wchar_t* json_message = (wchar_t*)(msg->lpData);
         if (webview_control != NULL) {
-          webview_control.InvokeScriptAsync(hstring(L"receive_from_settings_app"), { hstring(json_message) });
+          const auto _ = webview_control.InvokeScriptAsync(hstring(L"receive_from_settings_app"), { hstring(json_message) });
```

Comment:
What are those `_` for?

Reply:
That is a way to ignore the returned value and avoid this warning https://docs.microsoft.com/en-us/visualstudio/code-quality/c26444?view=vs-2019